### PR TITLE
[blog] add nette/symfony decorator

### PR DESCRIPTION
**Pull request by TomasVotruba**

### CODE REVIEW

File: source/_posts/2017/2017-02-12-drop-all-service-tags-in-your-nette-and-symfony-applications.md [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+```
+
+
+
+I recommend putting Decorator setup to `app/config/decorator.neon` or `app/config/decorator.yml`, so it's easy to
+find and programmer know about it.
+
+### Symfony Decorator Autowire Minitip
+
+Autowiring is on by default in Nette, but in Symfony you have to enabled it manually. You can use
```
Comment (enumag) [inline source/_posts/2017/2017-02-12-drop-all-service-tags-in-your-nette-and-symfony-applications.md]: ...have to enable it...

**Comment by TomasVotruba** [comment]
That feature is adding more unnecessary complexity with not added value. This will lead to programing in configs.

Would you consider trying this and letting me know the feedback?

**Comment by TomasVotruba** [comment]
Thanks for fixes!

**Comment by enumag** [comment]
> Would you consider trying this and letting me know the feedback?

I'm afraid it would mean too many changes in my codebase.

**Comment by TomasVotruba** [comment]
@enumag What changes exactly?

**Comment by enumag** [comment]
@TomasVotruba I'm using tags in many cases now, in many of my libraries. Changing it would mean to update nearly all of the libraries somehow (and in one case I don't think it's even possible) and then all my projects because it would be a BC break. Way too much work for an experiment.

**Comment by TomasVotruba** [comment]
@enumag That sounds like refactoring. Imagine you'd trying to show me new IDE and told me to use it everywhere from now on. That's too much.

Experiment has to be small (the smallest change possible that you can evaluate), in controlled environment.

It can be something like that: "I will change it for 3 commands in ProductModule".

That's the only way you can evaluate and judge based on your real personal evidence and not your imagination.

File: source/_posts/2017/2017-02-12-drop-all-service-tags-in-your-nette-and-symfony-applications.md [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+services:
+- { class: App\Console\FirstCommand, tags: [kdyby.console.command] }
+```
+
+This is what I did, before I used Decorator and before I dropped tags from my coding habbits.
+
+
+## Get Rig of Tagging in Symfony
+
+Symfony has very [developed system of service tagging](http://symfony.com/doc/current/reference/dic_tags.html) that is coupled to many internal parts. This is barely half of it:
```
Comment (TomasVotruba) [inline source/_posts/2017/2017-02-12-drop-all-service-tags-in-your-nette-and-symfony-applications.md]: In meaning of "many". I rephrased it.

**Pull request merged**